doc/source/notebooks/advanced/mcmc.pct.py

```diff
@@ -612,6 +612,9 @@ def run_chain_fn():
 # %% [markdown]
 # When a prior is set in *constrained* space, the `gpflow.optimizers.SamplingHelper` compensates the prior density with a Jacobian of the transformation used by the parameter. For priors in the `unconstrained` space the `gpflow.optimizers.SamplingHelper` will not apply adjustments to the prior density.
 
+# %% [markdown]
+# Below we repeat the same experiment that we ran in the beginning, but with some priors in `unconstrained` space:
```

Comment:
```suggestion
# Below we repeat the same experiment that we ran in the beginning, but with some priors in defined in the `unconstrained` space. Since we are using an Exp transform, a lognormal prior on a constrained parameter corresponds to a Normal prior on the unconstrained space:
```